### DISCUSSION TITLE
Very new to this thing

### DISCUSSION THREAD

**Discussion by Gondrong1211** [Discord-OTP-Forcer/Discord-OTP-Forcer#82] [Community Support]
Hi, I'm very new to this python and since im not coming from IT background, it is hard for me to understand the instructions on how to start it. (since I desperately want my discord account back due to the 2FA lockout). I've downloaded the 3.10 version of python.

There are few questions:

1. Where should I run the "scripts/Windows/setup.bat" ? Is it from the Python that has been installed or can i just click the "main.py" on the folder? I clicked the main.py but it shows nothing.
2. 
![image](https://github.com/Derpitron/Discord-OTP-Forcer/assets/147261498/387a01ab-7f3b-4fb5-bdbc-8d159b6ff337)
Is the token after the "=" is the same as my account's token?
3. The main.py is not working if i clicked it. Should i have to update it to the latest version?

Thank you very much!

**Comment by jurbek** [comment]
> 3. I _may_ have gotten a succesful TOTP code login a week ago but idiotically closed the browser window before saving the account 😬. Your mileage may vary

Hey, could you please elaborate on this one? so you accessed your lost account for sure, you saw your chats/servers etc? or you just saw that the 'TOTP worked' in logs? 

I'm one month in... Need some spark of hope =D

**Reply by jurbek** [reply]
well as for logs, I got one success too but it happened just after I closed the automated browser window unintentionally. So is it possible your browser crashed and only then you got $_success_$ ?

![image](https://github.com/Derpitron/Discord-OTP-Forcer/assets/165653271/fee8bd1c-3fa2-49a7-ad78-8bf07eff59e8)

**Reply by jurbek** [reply]
I didn't see anything in the browser window since I had closed it (by mistake, it wasn't a crash) and then immediately got the success in the console
